.gitignore

```diff
@@ -132,3 +132,6 @@ dmypy.json
 
 # visual studio
 .vscode/
+
+# mac files
+.DS_Store
```

Comment:
This (and even the `.vscode/` above) I really suggest removing from here, and put in your system (actually user) gitignore.
That is the correct level to put this information, since these files are not generated by tools used by the project, but by tools present on your system (thus it is overhead for the project, and it should be repeated for every project).

Moreover, there are still 3 `.DS_Store` files checked in:
![image](https://user-images.githubusercontent.com/10830876/171400693-82ae7131-117a-4d76-9061-3a3545d96c48.png)

Reply:
I am doing that now

Reply:
Ok solved now